XMonad/Layout/IndependentScreens.hs

```diff
@@ -276,6 +274,14 @@ unmarshallWindowSpace ws = ws { W.tag = unmarshallW (W.tag ws) }
 -- >    dynamicLogWithPP $ marshallPP s myPP {ppHidden = check, ppOutput = hPutStrLn h}
 -- >
 -- > main = do
--- >    h <- spawnPipe "xmobar"
--- >    h1 <- spawnPipe "xmobar"
+-- >    h <- spawnPipe "xmobar -x 0 <path to config>"
+-- >    h1 <- spawnPipe "xmobar -x 1 <path to config>"
 -- >    xmonad def { logHook = sampleLogHook h 0 <+> sampleLogHook h1 1 }
+
```

Comment:
Perhaps we should show the generalized form that works for an arbitrary number of screens, rather than its specialization to two screens. Something like this (untested):

    main = do
        n <- countScreens
        hs <- traverse (\(S n) -> spawnPipe ("xmobar -x " ++ show n ++ " ~/.xmobarrc-" ++ show n)) [0..n]
        xmonad def { logHook = mconcat (zipWith sampleLogHook hs [0..n]) }

Reply:
Okay, rereading, probably using `i` instead of `n` in the lambda is better, and both `[0..n]`'s should be `[0..n-1]`'s.

Reply:
Thanks for your suggestion. Have added it to the document and tested it.